docs/2.7.0/docs/daml-finance/tutorials/getting-started/holding.rst

```diff
@@ -0,0 +1,194 @@
+In particular, `An Introduction to Daml <https://docs.daml.com/daml/intro/0_Intro.html>`_
+would be a good starting point.
+
+This example refers to the :doc:`Transfer <transfer>` tutorial script in the sense that it explains
+the accounts and holdings required for that tutorial.
+
+Explanation of Holdings
+***********************
+
+As described in the :ref:`Asset model <holding-asset-model>`, a holding contract represents the
+ownership of a certain amount of an :ref:`instrument <instrument>` by an owner at a custodian.
+
+A holding implementation can have specific properties such as being :ref:`fungible <fungibility>` or
+:ref:`transferable <transferability>`.
+
+Instruments vs Holdings
+=======================
+
+* An instrument defines *what* a party holds (the rights and obligations).
+* A holding defines *how much* (ie., the amount) of an instrument and *against which party* (ie.,
+  the custodian) the instrument is being held.
+
+Bearer tokens vs Holdings
+=========================
+
+* Bearer tokens mean that *any* party in possession of the token is presumed to be the owner of the
+  asset.
+* Holdings, on the other hand, *restrict* the ownership of the asset to parties that have an account
+  (a vetted relationship) at a custodian.
+
+Sample code
+***********
+
+In order to show how this works in practice, let us now explore the ``Transfer`` script (used in the
+:doc:`Transfer <transfer>` chapter) step-by-step.
+
+Create ``Holding`` and ``Account`` Factories
+============================================
+
+The first instruction instantiates an account factory. This is just a template that is used by a
+party (the Bank in this case) to create accounts as part of the ``CreateAccount`` workflow.
+
+.. literalinclude:: /_templates/quickstart-finance/daml/Scripts/Holding.daml
+  :language: daml
+  :start-after: -- CREATE_ACCOUNT_FACTORY_BEGIN
+  :end-before: -- CREATE_ACCOUNT_FACTORY_END
+
+Notice how the ``ContractId`` is immediately converted to an interface upon creation: this is
+because our workflows do not have any knowledge of concrete template implementations.
+
+Similarly, we define a holding factory, which is used within an account to ``Credit`` and ``Debit``
+holdings.
+
+.. literalinclude:: /_templates/quickstart-finance/daml/Scripts/Holding.daml
+  :language: daml
+  :start-after: -- CREATE_HOLDING_FACTORY_BEGIN
+  :end-before: -- CREATE_HOLDING_FACTORY_END
+
+This factory contract can be used to create
+:ref:`Fungible <type-daml-finance-holding-fungible-factory-35358>` holdings, which are defined in
+:ref:`Daml.Finance.Holding.Fungible <module-daml-finance-holding-fungible-7201>`
+and are both :ref:`fungible <fungibility>`, as well as :ref:`transferable <transferability>`.
+
+We are adding a so-called *public party* as an observer to the holding factory. This is done to
+ensure that every other party has visibility over this contract, as all parties can `readAs` the
```

Comment:
I would reformulate this to: "This is done to ensure that parties, which have `readAs` rights of the public party, have visibility over the contract."